samples/bluetooth/peripheral_hr/sample.yaml

```diff
@@ -4,10 +4,5 @@ sample:
 tests:
   test:
     harness: bluetooth
-    platform_whitelist: qemu_cortex_m3 qemu_x86
-    tags: bluetooth
-  test.overlay-shield_frdm_kw41z:
-    harness: bluetooth
-    extra_args: OVERLAY_CONFIG=overlay-shield_frdm_kw41z.conf
-    platform_whitelist: mimxrt1020_evk mimxrt1050_evk mimxrt1060_evk frdm_k64f
+    platform_whitelist: qemu_cortex_m3 qemu_x86 mimxrt1020_evk mimxrt1050_evk mimxrt1060_evk frdm_k64f
```

Comment:
Instead of the modification to CMakeLists.txt, could you leave the two tests and set `extra_args: SHIELD=frdm_kw41` in the second one?

Reply:
I went ahead and merged this as I wanted to close on the base -DSHIELD support this week.  I'll open a new PR for reworking the bluetooth test as you've asked in here.

Reply:
I opened PR #12585 for the BT Periph Sample change.